docs/src/routes/(docs)/components/accordion/+page.svelte

```diff
@@ -347,20 +347,19 @@
         <h2>Bijbehorende bestanden</h2>
         <p>
           Voor meer informatie over importeren en instellen van componenten. Zie:
-          <a href="{base}/documentation/import-styling">
-            Componenten gebruiken en styling toevoegen
-          </a>
+          <a href="{base}/documentation/import-styling"
+            >Componenten gebruiken en styling toevoegen</a
+          >
         </p>
-        <h3>Benodigd</h3>
-        <ul>
-          <li><code>accordion.scss</code></li>
-          <li><code>accordion.js</code></li>
-        </ul>
 
-        <h3>Optioneel</h3>
-        <ul>
-          <li><code>accordion-variables.scss</code></li>
-        </ul>
+        <h3>Importeer component via NPM</h3>
```

Comment:
Nitpick: npm vindt zelf dat npm altijd in kleine letters moet.

Reply:
Mee eens. Goede suggestie. Ik zal het in een losse pr oppakken.

Reply:
Ticket: https://github.com/minvws/nl-rdo-manon/issues/624